test/e2e/set/scyllacluster/scyllacluster_tls.go

```diff
@@ -244,6 +249,128 @@ var _ = g.Describe("ScyllaCluster", func() {
 		}
 	})
 
+	g.It("should rotate TLS certificate without disrupting ongoing CQL traffic", func() {
```

Comment:
can this be tested inside `should rotate TLS certificates before they expire`?

Reply:
why these two should be coupled?

Reply:
each scyllacluster takes a lot of time to rollout and the certs are already available in the other test so this can fluently continue where the setup test has finished and save resources + execution time for the suite